src/Sulu/Bundle/PageBundle/Admin/PageAdmin.php

```diff
@@ -159,7 +159,7 @@ public function configureViews(ViewCollection $viewCollection): void
         ];
 
         $routerAttributesToFormRequest = ['parentId', 'webspace'];
-        $routerAttributesToFormMetdata = ['webspace'];
+        $routerAttributesToFormMetdata = ['webspace', 'defaultTemplate'];
```

Comment:
This will fragment the cache far more than it is necessary, that is why I suggested [in my comment](https://github.com/sulu/sulu/pull/5337#issuecomment-641900742) to implement this in a different way. Here we will have a metadata request for every `defaultTemplate` that is configured, whereby with the other solution it would always be just a single metadata request.